docs/architecture.md

```diff
@@ -107,6 +107,9 @@ The supported KV stores for the HA tracker are:
 
 * [Consul](https://www.consul.io)
 * [Etcd](https://etcd.io)
+* inmemory (only works if there is a single distributor)
```

Comment:
I would remove this because it's not correct. Inmemory only works when there's a single process (distributor and ingester must run in the same process) and it's basically just used for testing purposes. I believe documenting it may cause more misunderstanding.